Update npm dependencies, switch to lab

Siliconrob opened the pull request:
### Getting errors pulling down the npm package for `hapi-cron`.

Updated the dependencies and switched from `jest` (which seems to be installed globally) was not working over to `@hapi/lab` which is awesome and has no dependencies and your tests ran without issue


### Details from package download listed below
```
0 info it worked if it ends with ok
1 verbose cli [
1 verbose cli   '/usr/local/Cellar/node/13.7.0/bin/node',
1 verbose cli   '/usr/local/bin/npm',
1 verbose cli   'install',
1 verbose cli   '--save',
1 verbose cli   'hapi-cron'
1 verbose cli ]
2 info using npm@6.13.6
3 info using node@v13.7.0
4 verbose npm-session d173b094915e36e6
5 silly install loadCurrentTree
6 silly install readLocalPackageData
7 http fetch GET 200 https://registry.npmjs.org/hapi-cron 26ms (from cache)
8 silly pacote tag manifest for hapi-cron@latest fetched in 46ms
9 timing stage:loadCurrentTree Completed in 1044ms
10 silly install loadIdealTree
11 silly install cloneCurrentTreeToIdealTree
12 timing stage:loadIdealTree:cloneCurrentTree Completed in 6ms
13 silly install loadShrinkwrap
14 timing stage:rollbackFailedOptional Completed in 0ms
15 timing stage:runTopLevelLifecycles Completed in 1056ms
16 silly saveTree ercot-market@0.0.1
16 silly saveTree ├─┬ @hapi/hapi@18.4.0
16 silly saveTree │ ├─┬ @hapi/accept@3.2.3
16 silly saveTree │ │ ├─┬ @hapi/boom@7.4.11
16 silly saveTree │ │ │ └── @hapi/hoek@8.5.0
16 silly saveTree │ │ └── @hapi/hoek@8.5.0
16 silly saveTree │ ├── @hapi/ammo@3.1.1
16 silly saveTree │ ├── @hapi/boom@7.4.11
16 silly saveTree │ ├── @hapi/bounce@1.3.2
16 silly saveTree │ ├── @hapi/call@5.1.2
16 silly saveTree │ ├── @hapi/catbox-memory@4.1.1
16 silly saveTree │ ├─┬ @hapi/catbox@10.2.3
16 silly saveTree │ │ ├─┬ @hapi/joi@16.1.8
16 silly saveTree │ │ │ ├── @hapi/address@2.1.4
16 silly saveTree │ │ │ ├── @hapi/formula@1.2.0
16 silly saveTree │ │ │ ├── @hapi/pinpoint@1.0.2
16 silly saveTree │ │ │ └── @hapi/topo@3.1.6
16 silly saveTree │ │ └─┬ @hapi/podium@3.4.3
16 silly saveTree │ │   └── @hapi/joi@16.1.8
16 silly saveTree │ ├─┬ @hapi/heavy@6.2.2
16 silly saveTree │ │ └── @hapi/joi@16.1.8
16 silly saveTree │ ├── @hapi/hoek@8.5.0
16 silly saveTree │ ├─┬ @hapi/joi@15.1.1
16 silly saveTree │ │ └── @hapi/bourne@1.3.2
16 silly saveTree │ ├─┬ @hapi/mimos@4.1.1
16 silly saveTree │ │ └── mime-db@1.42.0
16 silly saveTree │ ├── @hapi/podium@3.4.3
16 silly saveTree │ ├─┬ @hapi/shot@4.1.2
16 silly saveTree │ │ └── @hapi/joi@16.1.8
16 silly saveTree │ ├── @hapi/somever@2.1.1
16 silly saveTree │ ├─┬ @hapi/statehood@6.1.2
16 silly saveTree │ │ ├── @hapi/cryptiles@4.2.1
16 silly saveTree │ │ ├─┬ @hapi/iron@5.1.4
16 silly saveTree │ │ │ └── @hapi/b64@4.2.1
16 silly saveTree │ │ └── @hapi/joi@16.1.8
16 silly saveTree │ ├─┬ @hapi/subtext@6.1.2
16 silly saveTree │ │ ├── @hapi/content@4.1.0
16 silly saveTree │ │ ├── @hapi/file@1.0.0
16 silly saveTree │ │ ├─┬ @hapi/pez@4.1.1
16 silly saveTree │ │ │ └─┬ @hapi/nigel@3.1.1
16 silly saveTree │ │ │   └── @hapi/vise@3.1.1
16 silly saveTree │ │ └── @hapi/wreck@15.1.0
16 silly saveTree │ ├── @hapi/teamwork@3.3.1
16 silly saveTree │ └── @hapi/topo@3.1.6
16 silly saveTree ├── @hapi/joi@16.1.8
16 silly saveTree ├─┬ async-csv@2.1.3
16 silly saveTree │ └─┬ csv@5.3.1
16 silly saveTree │   ├── csv-generate@3.2.4
16 silly saveTree │   ├── csv-parse@4.8.3
16 silly saveTree │   ├── csv-stringify@5.3.6
16 silly saveTree │   └─┬ stream-transform@2.0.1
16 silly saveTree │     └── mixme@0.3.5
16 silly saveTree ├─┬ axios@0.19.0
16 silly saveTree │ ├─┬ follow-redirects@1.5.10
16 silly saveTree │ │ └─┬ debug@3.1.0
16 silly saveTree │ │   └── ms@2.0.0
16 silly saveTree │ └── is-buffer@2.0.4
16 silly saveTree ├─┬ content-disposition@0.5.3
16 silly saveTree │ └── safe-buffer@5.1.2
16 silly saveTree ├── csv@5.3.1
16 silly saveTree ├─┬ csvtojson@2.0.10
16 silly saveTree │ ├── bluebird@3.7.2
16 silly saveTree │ ├── lodash@4.17.15
16 silly saveTree │ └─┬ strip-bom@2.0.0
16 silly saveTree │   └── is-utf8@0.2.1
16 silly saveTree ├─┬ fs-extra@8.1.0
16 silly saveTree │ ├── graceful-fs@4.2.3
16 silly saveTree │ ├── jsonfile@4.0.0
16 silly saveTree │ └── universalify@0.1.2
16 silly saveTree ├─┬ hapi-auto-route@3.0.4
16 silly saveTree │ ├── @hapi/joi@15.1.1
16 silly saveTree │ └─┬ glob@7.1.6
16 silly saveTree │   ├── fs.realpath@1.0.0
16 silly saveTree │   ├─┬ inflight@1.0.6
16 silly saveTree │   │ ├─┬ once@1.4.0
16 silly saveTree │   │ │ └── wrappy@1.0.2
16 silly saveTree │   │ └── wrappy@1.0.2
16 silly saveTree │   ├── inherits@2.0.4
16 silly saveTree │   ├─┬ minimatch@3.0.4
16 silly saveTree │   │ └─┬ brace-expansion@1.1.11
16 silly saveTree │   │   ├── balanced-match@1.0.0
16 silly saveTree │   │   └── concat-map@0.0.1
16 silly saveTree │   ├── once@1.4.0
16 silly saveTree │   └── path-is-absolute@1.0.1
16 silly saveTree ├─┬ jssoup@0.0.10
16 silly saveTree │ ├─┬ debug@2.6.9
16 silly saveTree │ │ └── ms@2.0.0
16 silly saveTree │ ├── growl@1.10.5
16 silly saveTree │ └── htmlparser@1.7.7
16 silly saveTree ├─┬ knex@0.20.6
16 silly saveTree │ ├── colorette@1.1.0
16 silly saveTree │ ├── commander@4.0.1
16 silly saveTree │ ├─┬ debug@4.1.1
16 silly saveTree │ │ └── ms@2.1.2
16 silly saveTree │ ├── getopts@2.2.5
16 silly saveTree │ ├── interpret@2.0.0
16 silly saveTree │ ├─┬ liftoff@3.1.0
16 silly saveTree │ │ ├── extend@3.0.2
16 silly saveTree │ │ ├─┬ findup-sync@3.0.0
16 silly saveTree │ │ │ ├── detect-file@1.0.0
16 silly saveTree │ │ │ ├─┬ is-glob@4.0.1
16 silly saveTree │ │ │ │ └── is-extglob@2.1.1
16 silly saveTree │ │ │ ├─┬ micromatch@3.1.10
16 silly saveTree │ │ │ │ ├── arr-diff@4.0.0
16 silly saveTree │ │ │ │ ├── array-unique@0.3.2
16 silly saveTree │ │ │ │ ├─┬ braces@2.3.2
16 silly saveTree │ │ │ │ │ ├── arr-flatten@1.1.0
16 silly saveTree │ │ │ │ │ ├─┬ extend-shallow@2.0.1
16 silly saveTree │ │ │ │ │ │ └── is-extendable@0.1.1
16 silly saveTree │ │ │ │ │ ├─┬ fill-range@4.0.0
16 silly saveTree │ │ │ │ │ │ ├── extend-shallow@2.0.1
16 silly saveTree │ │ │ │ │ │ ├─┬ is-number@3.0.0
16 silly saveTree │ │ │ │ │ │ │ └─┬ kind-of@3.2.2
16 silly saveTree │ │ │ │ │ │ │   └── is-buffer@1.1.6
16 silly saveTree │ │ │ │ │ │ ├── repeat-string@1.6.1
16 silly saveTree │ │ │ │ │ │ └── to-regex-range@2.1.1
16 silly saveTree │ │ │ │ │ ├── isobject@3.0.1
16 silly saveTree │ │ │ │ │ ├── repeat-element@1.1.3
16 silly saveTree │ │ │ │ │ ├─┬ snapdragon-node@2.1.1
16 silly saveTree │ │ │ │ │ │ ├─┬ define-property@1.0.0
16 silly saveTree │ │ │ │ │ │ │ └─┬ is-descriptor@1.0.2
16 silly saveTree │ │ │ │ │ │ │   ├─┬ is-accessor-descriptor@1.0.0
16 silly saveTree │ │ │ │ │ │ │   │ └── kind-of@6.0.2
16 silly saveTree │ │ │ │ │ │ │   ├── is-data-descriptor@1.0.0
16 silly saveTree │ │ │ │ │ │ │   └── kind-of@6.0.2
16 silly saveTree │ │ │ │ │ │ └─┬ snapdragon-util@3.0.1
16 silly saveTree │ │ │ │ │ │   └── kind-of@3.2.2
16 silly saveTree │ │ │ │ │ ├─┬ snapdragon@0.8.2
16 silly saveTree │ │ │ │ │ │ ├─┬ base@0.11.2
16 silly saveTree │ │ │ │ │ │ │ ├─┬ cache-base@1.0.1
16 silly saveTree │ │ │ │ │ │ │ │ ├─┬ collection-visit@1.0.0
16 silly saveTree │ │ │ │ │ │ │ │ │ ├─┬ map-visit@1.0.0
16 silly saveTree │ │ │ │ │ │ │ │ │ │ └── object-visit@1.0.1
16 silly saveTree │ │ │ │ │ │ │ │ │ └── object-visit@1.0.1
16 silly saveTree │ │ │ │ │ │ │ │ ├── component-emitter@1.3.0
16 silly saveTree │ │ │ │ │ │ │ │ ├── get-value@2.0.6
16 silly saveTree │ │ │ │ │ │ │ │ ├─┬ has-value@1.0.0
16 silly saveTree │ │ │ │ │ │ │ │ │ └─┬ has-values@1.0.0
16 silly saveTree │ │ │ │ │ │ │ │ │   └── kind-of@4.0.0
16 silly saveTree │ │ │ │ │ │ │ │ ├─┬ set-value@2.0.1
16 silly saveTree │ │ │ │ │ │ │ │ │ ├── extend-shallow@2.0.1
16 silly saveTree │ │ │ │ │ │ │ │ │ ├── is-plain-object@2.0.4
16 silly saveTree │ │ │ │ │ │ │ │ │ └─┬ split-string@3.1.0
16 silly saveTree │ │ │ │ │ │ │ │ │   └─┬ extend-shallow@3.0.2
16 silly saveTree │ │ │ │ │ │ │ │ │     ├── assign-symbols@1.0.0
16 silly saveTree │ │ │ │ │ │ │ │ │     └── is-extendable@1.0.1
16 silly saveTree │ │ │ │ │ │ │ │ ├─┬ to-object-path@0.3.0
16 silly saveTree │ │ │ │ │ │ │ │ │ └── kind-of@3.2.2
16 silly saveTree │ │ │ │ │ │ │ │ ├─┬ union-value@1.0.1
16 silly saveTree │ │ │ │ │ │ │ │ │ └── arr-union@3.1.0
16 silly saveTree │ │ │ │ │ │ │ │ └─┬ unset-value@1.0.0
16 silly saveTree │ │ │ │ │ │ │ │   └─┬ has-value@0.3.1
16 silly saveTree │ │ │ │ │ │ │ │     ├── has-values@0.1.4
16 silly saveTree │ │ │ │ │ │ │ │     └─┬ isobject@2.1.0
16 silly saveTree │ │ │ │ │ │ │ │       └── isarray@1.0.0
16 silly saveTree │ │ │ │ │ │ │ ├─┬ class-utils@0.3.6
16 silly saveTree │ │ │ │ │ │ │ │ ├─┬ define-property@0.2.5
16 silly saveTree │ │ │ │ │ │ │ │ │ └─┬ is-descriptor@0.1.6
16 silly saveTree │ │ │ │ │ │ │ │ │   ├─┬ is-accessor-descriptor@0.1.6
16 silly saveTree │ │ │ │ │ │ │ │ │   │ └── kind-of@3.2.2
16 silly saveTree │ │ │ │ │ │ │ │ │   ├─┬ is-data-descriptor@0.1.4
16 silly saveTree │ │ │ │ │ │ │ │ │   │ └── kind-of@3.2.2
16 silly saveTree │ │ │ │ │ │ │ │ │   └── kind-of@5.1.0
16 silly saveTree │ │ │ │ │ │ │ │ └─┬ static-extend@0.1.2
16 silly saveTree │ │ │ │ │ │ │ │   ├── define-property@0.2.5
16 silly saveTree │ │ │ │ │ │ │ │   └─┬ object-copy@0.1.0
16 silly saveTree │ │ │ │ │ │ │ │     ├── copy-descriptor@0.1.1
16 silly saveTree │ │ │ │ │ │ │ │     ├── define-property@0.2.5
16 silly saveTree │ │ │ │ │ │ │ │     └── kind-of@3.2.2
16 silly saveTree │ │ │ │ │ │ │ ├── component-emitter@1.3.0
16 silly saveTree │ │ │ │ │ │ │ ├─┬ define-property@1.0.0
16 silly saveTree │ │ │ │ │ │ │ │ └─┬ is-descriptor@1.0.2
16 silly saveTree │ │ │ │ │ │ │ │   ├── is-accessor-descriptor@1.0.0
16 silly saveTree │ │ │ │ │ │ │ │   └── is-data-descriptor@1.0.0
16 silly saveTree │ │ │ │ │ │ │ ├─┬ mixin-deep@1.3.2
16 silly saveTree │ │ │ │ │ │ │ │ ├── for-in@1.0.2
16 silly saveTree │ │ │ │ │ │ │ │ └── is-extendable@1.0.1
16 silly saveTree │ │ │ │ │ │ │ └── pascalcase@0.1.1
16 silly saveTree │ │ │ │ │ │ ├─┬ debug@2.6.9
16 silly saveTree │ │ │ │ │ │ │ └── ms@2.0.0
16 silly saveTree │ │ │ │ │ │ ├── define-property@0.2.5
16 silly saveTree │ │ │ │ │ │ ├── extend-shallow@2.0.1
16 silly saveTree │ │ │ │ │ │ ├── map-cache@0.2.2
16 silly saveTree │ │ │ │ │ │ ├─┬ source-map-resolve@0.5.3
16 silly saveTree │ │ │ │ │ │ │ ├── atob@2.1.2
16 silly saveTree │ │ │ │ │ │ │ ├── decode-uri-component@0.2.0
16 silly saveTree │ │ │ │ │ │ │ ├── resolve-url@0.2.1
16 silly saveTree │ │ │ │ │ │ │ ├── source-map-url@0.4.0
16 silly saveTree │ │ │ │ │ │ │ └── urix@0.1.0
16 silly saveTree │ │ │ │ │ │ ├── source-map@0.5.7
16 silly saveTree │ │ │ │ │ │ └── use@3.1.1
16 silly saveTree │ │ │ │ │ ├── split-string@3.1.0
16 silly saveTree │ │ │ │ │ └─┬ to-regex@3.0.2
16 silly saveTree │ │ │ │ │   ├─┬ define-property@2.0.2
16 silly saveTree │ │ │ │ │   │ └─┬ is-descriptor@1.0.2
16 silly saveTree │ │ │ │ │   │   ├── is-accessor-descriptor@1.0.0
16 silly saveTree │ │ │ │ │   │   └── is-data-descriptor@1.0.0
16 silly saveTree │ │ │ │ │   ├─┬ regex-not@1.0.2
16 silly saveTree │ │ │ │ │   │ └─┬ safe-regex@1.1.0
16 silly saveTree │ │ │ │ │   │   └── ret@0.1.15
16 silly saveTree │ │ │ │ │   └── safe-regex@1.1.0
16 silly saveTree │ │ │ │ ├── define-property@2.0.2
16 silly saveTree │ │ │ │ ├── extend-shallow@3.0.2
16 silly saveTree │ │ │ │ ├─┬ extglob@2.0.4
16 silly saveTree │ │ │ │ │ ├─┬ define-property@1.0.0
16 silly saveTree │ │ │ │ │ │ └─┬ is-descriptor@1.0.2
16 silly saveTree │ │ │ │ │ │   ├── is-accessor-descriptor@1.0.0
16 silly saveTree │ │ │ │ │ │   └── is-data-descriptor@1.0.0
16 silly saveTree │ │ │ │ │ ├─┬ expand-brackets@2.1.4
16 silly saveTree │ │ │ │ │ │ ├─┬ debug@2.6.9
16 silly saveTree │ │ │ │ │ │ │ └── ms@2.0.0
16 silly saveTree │ │ │ │ │ │ ├── define-property@0.2.5
16 silly saveTree │ │ │ │ │ │ ├── extend-shallow@2.0.1
16 silly saveTree │ │ │ │ │ │ └── posix-character-classes@0.1.1
16 silly saveTree │ │ │ │ │ ├── extend-shallow@2.0.1
16 silly saveTree │ │ │ │ │ └── fragment-cache@0.2.1
16 silly saveTree │ │ │ │ ├── fragment-cache@0.2.1
16 silly saveTree │ │ │ │ ├── kind-of@6.0.2
16 silly saveTree │ │ │ │ ├─┬ nanomatch@1.2.13
16 silly saveTree │ │ │ │ │ ├── is-windows@1.0.2
16 silly saveTree │ │ │ │ │ └── object.pick@1.3.0
16 silly saveTree │ │ │ │ ├── object.pick@1.3.0
16 silly saveTree │ │ │ │ ├── regex-not@1.0.2
16 silly saveTree │ │ │ │ ├── snapdragon@0.8.2
16 silly saveTree │ │ │ │ └── to-regex@3.0.2
16 silly saveTree │ │ │ └─┬ resolve-dir@1.0.1
16 silly saveTree │ │ │   ├─┬ expand-tilde@2.0.2
16 silly saveTree │ │ │   │ └─┬ homedir-polyfill@1.0.3
16 silly saveTree │ │ │   │   └── parse-passwd@1.0.0
16 silly saveTree │ │ │   └─┬ global-modules@1.0.0
16 silly saveTree │ │ │     └─┬ global-prefix@1.0.2
16 silly saveTree │ │ │       ├── ini@1.3.5
16 silly saveTree │ │ │       └─┬ which@1.3.1
16 silly saveTree │ │ │         └── isexe@2.0.0
16 silly saveTree │ │ ├─┬ fined@1.2.0
16 silly saveTree │ │ │ ├─┬ object.defaults@1.1.0
16 silly saveTree │ │ │ │ ├── array-each@1.0.1
16 silly saveTree │ │ │ │ ├── array-slice@1.1.0
16 silly saveTree │ │ │ │ └── for-own@1.0.0
16 silly saveTree │ │ │ └─┬ parse-filepath@1.0.2
16 silly saveTree │ │ │   ├─┬ is-absolute@1.0.0
16 silly saveTree │ │ │   │ └─┬ is-relative@1.0.0
16 silly saveTree │ │ │   │   └─┬ is-unc-path@1.0.0
16 silly saveTree │ │ │   │     └── unc-path-regex@0.1.2
16 silly saveTree │ │ │   └─┬ path-root@0.1.1
16 silly saveTree │ │ │     └── path-root-regex@0.1.2
16 silly saveTree │ │ ├── flagged-respawn@1.0.1
16 silly saveTree │ │ ├── is-plain-object@2.0.4
16 silly saveTree │ │ ├─┬ object.map@1.0.1
16 silly saveTree │ │ │ └── make-iterator@1.0.1
16 silly saveTree │ │ ├─┬ rechoir@0.6.2
16 silly saveTree │ │ │ └─┬ resolve@1.14.1
16 silly saveTree │ │ │   └── path-parse@1.0.6
16 silly saveTree │ │ └── resolve@1.14.1
16 silly saveTree │ ├─┬ mkdirp@0.5.1
16 silly saveTree │ │ └── minimist@0.0.8
16 silly saveTree │ ├── pg-connection-string@2.1.0
16 silly saveTree │ ├── tarn@2.0.0
16 silly saveTree │ ├── tildify@2.0.0
16 silly saveTree │ ├── uuid@3.3.3
16 silly saveTree │ └── v8flags@3.1.3
16 silly saveTree ├── luxon@1.22.0
16 silly saveTree ├─┬ sqlite3@4.1.1
16 silly saveTree │ ├── nan@2.14.0
16 silly saveTree │ ├─┬ node-pre-gyp@0.11.0
16 silly saveTree │ │ ├── detect-libc@1.0.3
16 silly saveTree │ │ ├─┬ needle@2.4.0
16 silly saveTree │ │ │ ├── debug@3.2.6
16 silly saveTree │ │ │ ├─┬ iconv-lite@0.4.24
16 silly saveTree │ │ │ │ └── safer-buffer@2.1.2
16 silly saveTree │ │ │ └── sax@1.2.4
16 silly saveTree │ │ ├─┬ nopt@4.0.1
16 silly saveTree │ │ │ ├── abbrev@1.1.1
16 silly saveTree │ │ │ └─┬ osenv@0.1.5
16 silly saveTree │ │ │   ├── os-homedir@1.0.2
16 silly saveTree │ │ │   └── os-tmpdir@1.0.2
16 silly saveTree │ │ ├─┬ npm-packlist@1.4.7
16 silly saveTree │ │ │ ├── ignore-walk@3.0.3
16 silly saveTree │ │ │ └─┬ npm-bundled@1.1.1
16 silly saveTree │ │ │   └── npm-normalize-package-bin@1.0.1
16 silly saveTree │ │ ├─┬ npmlog@4.1.2
16 silly saveTree │ │ │ ├─┬ are-we-there-yet@1.1.5
16 silly saveTree │ │ │ │ ├── delegates@1.0.0
16 silly saveTree │ │ │ │ └─┬ readable-stream@2.3.6
16 silly saveTree │ │ │ │   ├── core-util-is@1.0.2
16 silly saveTree │ │ │ │   ├── process-nextick-args@2.0.1
16 silly saveTree │ │ │ │   ├── string_decoder@1.1.1
16 silly saveTree │ │ │ │   └── util-deprecate@1.0.2
16 silly saveTree │ │ │ ├── console-control-strings@1.1.0
16 silly saveTree │ │ │ ├─┬ gauge@2.7.4
16 silly saveTree │ │ │ │ ├── aproba@1.2.0
16 silly saveTree │ │ │ │ ├── has-unicode@2.0.1
16 silly saveTree │ │ │ │ ├── object-assign@4.1.1
16 silly saveTree │ │ │ │ ├── signal-exit@3.0.2
16 silly saveTree │ │ │ │ ├─┬ string-width@1.0.2
16 silly saveTree │ │ │ │ │ ├── code-point-at@1.1.0
16 silly saveTree │ │ │ │ │ ├─┬ is-fullwidth-code-point@1.0.0
16 silly saveTree │ │ │ │ │ │ └── number-is-nan@1.0.1
16 silly saveTree │ │ │ │ │ └─┬ strip-ansi@3.0.1
16 silly saveTree │ │ │ │ │   └── ansi-regex@2.1.1
16 silly saveTree │ │ │ │ ├── strip-ansi@3.0.1
16 silly saveTree │ │ │ │ └── wide-align@1.1.3
16 silly saveTree │ │ │ └── set-blocking@2.0.0
16 silly saveTree │ │ ├─┬ rc@1.2.8
16 silly saveTree │ │ │ ├── deep-extend@0.6.0
16 silly saveTree │ │ │ ├── minimist@1.2.0
16 silly saveTree │ │ │ └── strip-json-comments@2.0.1
16 silly saveTree │ │ ├── rimraf@2.7.1
16 silly saveTree │ │ ├── semver@5.7.1
16 silly saveTree │ │ └─┬ tar@4.4.13
16 silly saveTree │ │   ├── chownr@1.1.3
16 silly saveTree │ │   ├─┬ fs-minipass@1.2.7
16 silly saveTree │ │   │ └─┬ minipass@2.9.0
16 silly saveTree │ │   │   └── yallist@3.1.1
16 silly saveTree │ │   ├── minipass@2.9.0
16 silly saveTree │ │   ├── minizlib@1.3.3
16 silly saveTree │ │   └── yallist@3.1.1
16 silly saveTree │ └─┬ request@2.88.0
16 silly saveTree │   ├── aws-sign2@0.7.0
16 silly saveTree │   ├── aws4@1.9.0
16 silly saveTree │   ├── caseless@0.12.0
16 silly saveTree │   ├─┬ combined-stream@1.0.8
16 silly saveTree │   │ └── delayed-stream@1.0.0
16 silly saveTree │   ├── forever-agent@0.6.1
16 silly saveTree │   ├─┬ form-data@2.3.3
16 silly saveTree │   │ ├── asynckit@0.4.0
16 silly saveTree │   │ └── mime-types@2.1.25
16 silly saveTree │   ├─┬ har-validator@5.1.3
16 silly saveTree │   │ ├─┬ ajv@6.10.2
16 silly saveTree │   │ │ ├── fast-deep-equal@2.0.1
16 silly saveTree │   │ │ ├── fast-json-stable-stringify@2.1.0
16 silly saveTree │   │ │ ├── json-schema-traverse@0.4.1
16 silly saveTree │   │ │ └─┬ uri-js@4.2.2
16 silly saveTree │   │ │   └── punycode@2.1.1
16 silly saveTree │   │ └── har-schema@2.0.0
16 silly saveTree │   ├─┬ http-signature@1.2.0
16 silly saveTree │   │ ├── assert-plus@1.0.0
16 silly saveTree │   │ ├─┬ jsprim@1.4.1
16 silly saveTree │   │ │ ├── extsprintf@1.3.0
16 silly saveTree │   │ │ ├── json-schema@0.2.3
16 silly saveTree │   │ │ └── verror@1.10.0
16 silly saveTree │   │ └─┬ sshpk@1.16.1
16 silly saveTree │   │   ├── asn1@0.2.4
16 silly saveTree │   │   ├─┬ bcrypt-pbkdf@1.0.2
16 silly saveTree │   │   │ └── tweetnacl@0.14.5
16 silly saveTree │   │   ├── dashdash@1.14.1
16 silly saveTree │   │   ├─┬ ecc-jsbn@0.1.2
16 silly saveTree │   │   │ └── jsbn@0.1.1
16 silly saveTree │   │   ├── getpass@0.1.7
16 silly saveTree │   │   ├── jsbn@0.1.1
16 silly saveTree │   │   └── tweetnacl@0.14.5
16 silly saveTree │   ├── is-typedarray@1.0.0
16 silly saveTree │   ├── isstream@0.1.2
16 silly saveTree │   ├── json-stringify-safe@5.0.1
16 silly saveTree │   ├── mime-types@2.1.25
16 silly saveTree │   ├── oauth-sign@0.9.0
16 silly saveTree │   ├── performance-now@2.1.0
16 silly saveTree │   ├── qs@6.5.2
16 silly saveTree │   ├─┬ tough-cookie@2.4.3
16 silly saveTree │   │ ├── psl@1.7.0
16 silly saveTree │   │ └── punycode@1.4.1
16 silly saveTree │   └── tunnel-agent@0.6.0
16 silly saveTree ├─┬ temp-write@4.0.0
16 silly saveTree │ ├── is-stream@2.0.0
16 silly saveTree │ ├─┬ make-dir@3.0.0
16 silly saveTree │ │ └── semver@6.3.0
16 silly saveTree │ └── temp-dir@1.0.0
16 silly saveTree └─┬ unzip-simple@1.0.4
16 silly saveTree   ├─┬ micromatch@4.0.2
16 silly saveTree   │ ├─┬ braces@3.0.2
16 silly saveTree   │ │ └─┬ fill-range@7.0.1
16 silly saveTree   │ │   └─┬ to-regex-range@5.0.1
16 silly saveTree   │ │     └── is-number@7.0.0
16 silly saveTree   │ └── picomatch@2.2.1
16 silly saveTree   └─┬ yauzl@2.10.0
16 silly saveTree     ├── buffer-crc32@0.2.13
16 silly saveTree     └─┬ fd-slicer@1.1.0
16 silly saveTree       └── pend@1.2.0
17 verbose stack SyntaxError: Unexpected end of JSON input while parsing near '...
17 verbose stack       }
17 verbose stack     }
17 verbose stack   }
17 verbose stack }
17 verbose stack '
17 verbose stack     at JSON.parse (<anonymous>)
17 verbose stack     at parseJson (/usr/local/lib/node_modules/npm/node_modules/json-parse-better-errors/index.js:7:17)
17 verbose stack     at module.exports (/usr/local/lib/node_modules/npm/lib/utils/parse-json.js:4:10)
17 verbose stack     at parsePkgLock (/usr/local/lib/node_modules/npm/lib/install/read-shrinkwrap.js:67:12)
17 verbose stack     at /usr/local/lib/node_modules/npm/lib/install/read-shrinkwrap.js:27:22
17 verbose stack     at tryCatcher (/usr/local/lib/node_modules/npm/node_modules/bluebird/js/release/util.js:16:23)
17 verbose stack     at Holder$2._callFunction (eval at generateHolderClass (/usr/local/lib/node_modules/npm/node_modules/bluebird/js/release/join.js:92:16), <anonymous>:12:44)
17 verbose stack     at Holder$2.checkFulfillment (eval at generateHolderClass (/usr/local/lib/node_modules/npm/node_modules/bluebird/js/release/join.js:92:16), <anonymous>:27:30)
17 verbose stack     at Promise.eval (eval at thenCallback (/usr/local/lib/node_modules/npm/node_modules/bluebird/js/release/join.js:14:16), <anonymous>:4:20)
17 verbose stack     at Promise._settlePromise (/usr/local/lib/node_modules/npm/node_modules/bluebird/js/release/promise.js:571:21)
17 verbose stack     at Promise._settlePromise0 (/usr/local/lib/node_modules/npm/node_modules/bluebird/js/release/promise.js:619:10)
17 verbose stack     at Promise._settlePromises (/usr/local/lib/node_modules/npm/node_modules/bluebird/js/release/promise.js:699:18)
17 verbose stack     at Promise._fulfill (/usr/local/lib/node_modules/npm/node_modules/bluebird/js/release/promise.js:643:18)
17 verbose stack     at Promise._settlePromise (/usr/local/lib/node_modules/npm/node_modules/bluebird/js/release/promise.js:587:21)
17 verbose stack     at Promise._settlePromise0 (/usr/local/lib/node_modules/npm/node_modules/bluebird/js/release/promise.js:619:10)
17 verbose stack     at Promise._settlePromises (/usr/local/lib/node_modules/npm/node_modules/bluebird/js/release/promise.js:699:18)
18 verbose cwd /Users/siliconrob/projects/ercot-market/test
19 verbose Darwin 19.3.0
20 verbose argv "/usr/local/Cellar/node/13.7.0/bin/node" "/usr/local/bin/npm" "install" "--save" "hapi-cron"
21 verbose node v13.7.0
22 verbose npm  v6.13.6
23 error Unexpected end of JSON input while parsing near '...
23 error       }
23 error     }
23 error   }
23 error }
23 error '
24 verbose exit [ 1, true ]
```

antonsamper commented:
@Siliconrob I havent come across this error before. What OS, node and npm versions are you using?

Siliconrob commented:
- OS: `Darwin Vault13.local 19.3.0 Darwin Kernel Version 19.3.0: Thu Jan  9 20:58:23 PST 2020; root:xnu-6153.81.5~1/RELEASE_X86_64 x86_6`
- Node: `v13.8.0`
- NPM: `6.13.7`

antonsamper commented:
I’ve tried the same versions and I’m not seeing any issues. It would be good if you can find/investigate this further without swapping out the testing framework.

Jest wont be causing any issues as it’s a devDependency so won’t be getting installed when you run:

```bash
$ npm i hapi-cron
```

Siliconrob commented:
Ok will try again.

Did you see that the `hapi` and `cron` dependencies this project are bound to are out of date as well and are being updated here?

antonsamper commented:
Happy to accept the dependency updates

Siliconrob commented:
Here you go on windows machine that never had the package installed
- OS: `Microsoft Windows [Version 10.0.18363.657]`
- Node: `v13.6.0`
- NPM: `6.13.4`

Your fork 
```
C:\projects\hapi-cron>npm install
npm WARN deprecated @hapi/hoek@7.2.1: This version has been deprecated in accordance with the hapi support policy (hapi.im/support). Please upgrade to the latest version to get the best features, bug fixes, and security patches. If you are unable to upgrade at this time, paid support is available for older versions (hapi.im/commercial).
npm WARN deprecated left-pad@1.3.0: use String.prototype.padStart()
npm WARN deprecated request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142
npm notice created a lockfile as package-lock.json. You should commit this file.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@^1.2.7 (node_modules\jest-haste-map\node_modules\fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.11: wanted {"os":"darwin","arch":"any"} (current: {"os":"win32","arch":"x64"})
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: abbrev@1.1.1 (node_modules\fsevents\node_modules\abbrev):
npm WARN enoent SKIPPING OPTIONAL DEPENDENCY: ENOENT: no such file or directory, rename 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\abbrev' -> 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\.abbrev.DELETE'
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: ansi-regex@2.1.1 (node_modules\fsevents\node_modules\ansi-regex):
npm WARN enoent SKIPPING OPTIONAL DEPENDENCY: ENOENT: no such file or directory, rename 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\ansi-regex' -> 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\.ansi-regex.DELETE'
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: aproba@1.2.0 (node_modules\fsevents\node_modules\aproba):
npm WARN enoent SKIPPING OPTIONAL DEPENDENCY: ENOENT: no such file or directory, rename 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\aproba' -> 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\.aproba.DELETE'
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: balanced-match@1.0.0 (node_modules\fsevents\node_modules\balanced-match):
npm WARN enoent SKIPPING OPTIONAL DEPENDENCY: ENOENT: no such file or directory, rename 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\balanced-match' -> 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\.balanced-match.DELETE'
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: chownr@1.1.3 (node_modules\fsevents\node_modules\chownr):
npm WARN enoent SKIPPING OPTIONAL DEPENDENCY: ENOENT: no such file or directory, rename 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\chownr' -> 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\.chownr.DELETE'
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: code-point-at@1.1.0 (node_modules\fsevents\node_modules\code-point-at):
npm WARN enoent SKIPPING OPTIONAL DEPENDENCY: ENOENT: no such file or directory, rename 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\code-point-at' -> 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\.code-point-at.DELETE'
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: concat-map@0.0.1 (node_modules\fsevents\node_modules\concat-map):
npm WARN enoent SKIPPING OPTIONAL DEPENDENCY: ENOENT: no such file or directory, rename 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\concat-map' -> 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\.concat-map.DELETE'
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: console-control-strings@1.1.0 (node_modules\fsevents\node_modules\console-control-strings):
npm WARN enoent SKIPPING OPTIONAL DEPENDENCY: ENOENT: no such file or directory, rename 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\console-control-strings' -> 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\.console-control-strings.DELETE'
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: core-util-is@1.0.2 (node_modules\fsevents\node_modules\core-util-is):
npm WARN enoent SKIPPING OPTIONAL DEPENDENCY: ENOENT: no such file or directory, rename 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\core-util-is' -> 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\.core-util-is.DELETE'
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: deep-extend@0.6.0 (node_modules\fsevents\node_modules\deep-extend):
npm WARN enoent SKIPPING OPTIONAL DEPENDENCY: ENOENT: no such file or directory, rename 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\deep-extend' -> 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\.deep-extend.DELETE'npm WARN optional SKIPPING OPTIONAL DEPENDENCY: delegates@1.0.0 (node_modules\fsevents\node_modules\delegates):
npm WARN enoent SKIPPING OPTIONAL DEPENDENCY: ENOENT: no such file or directory, rename 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\delegates' -> 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\.delegates.DELETE'
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: detect-libc@1.0.3 (node_modules\fsevents\node_modules\detect-libc):
npm WARN enoent SKIPPING OPTIONAL DEPENDENCY: ENOENT: no such file or directory, rename 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\detect-libc' -> 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\.detect-libc.DELETE'npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fs.realpath@1.0.0 (node_modules\fsevents\node_modules\fs.realpath):
npm WARN enoent SKIPPING OPTIONAL DEPENDENCY: ENOENT: no such file or directory, rename 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\fs.realpath' -> 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\.fs.realpath.DELETE'npm WARN optional SKIPPING OPTIONAL DEPENDENCY: has-unicode@2.0.1 (node_modules\fsevents\node_modules\has-unicode):
npm WARN enoent SKIPPING OPTIONAL DEPENDENCY: ENOENT: no such file or directory, rename 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\has-unicode' -> 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\.has-unicode.DELETE'npm WARN optional SKIPPING OPTIONAL DEPENDENCY: inherits@2.0.4 (node_modules\fsevents\node_modules\inherits):
npm WARN enoent SKIPPING OPTIONAL DEPENDENCY: ENOENT: no such file or directory, rename 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\inherits' -> 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\.inherits.DELETE'
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: ini@1.3.5 (node_modules\fsevents\node_modules\ini):
npm WARN enoent SKIPPING OPTIONAL DEPENDENCY: ENOENT: no such file or directory, rename 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\ini' -> 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\.ini.DELETE'
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: isarray@1.0.0 (node_modules\fsevents\node_modules\isarray):
npm WARN enoent SKIPPING OPTIONAL DEPENDENCY: ENOENT: no such file or directory, rename 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\isarray' -> 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\.isarray.DELETE'
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: minimist@0.0.8 (node_modules\fsevents\node_modules\minimist):
npm WARN enoent SKIPPING OPTIONAL DEPENDENCY: ENOENT: no such file or directory, rename 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\minimist' -> 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\.minimist.DELETE'
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: ms@2.1.2 (node_modules\fsevents\node_modules\ms):
npm WARN enoent SKIPPING OPTIONAL DEPENDENCY: ENOENT: no such file or directory, rename 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\ms' -> 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\.ms.DELETE'
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: npm-normalize-package-bin@1.0.1 (node_modules\fsevents\node_modules\npm-normalize-package-bin):
npm WARN enoent SKIPPING OPTIONAL DEPENDENCY: ENOENT: no such file or directory, rename 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\npm-normalize-package-bin' -> 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\.npm-normalize-package-bin.DELETE'
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: number-is-nan@1.0.1 (node_modules\fsevents\node_modules\number-is-nan):
npm WARN enoent SKIPPING OPTIONAL DEPENDENCY: ENOENT: no such file or directory, rename 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\number-is-nan' -> 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\.number-is-nan.DELETE'
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: object-assign@4.1.1 (node_modules\fsevents\node_modules\object-assign):
npm WARN enoent SKIPPING OPTIONAL DEPENDENCY: ENOENT: no such file or directory, rename 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\object-assign' -> 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\.object-assign.DELETE'
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: os-homedir@1.0.2 (node_modules\fsevents\node_modules\os-homedir):
npm WARN enoent SKIPPING OPTIONAL DEPENDENCY: ENOENT: no such file or directory, rename 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\os-homedir' -> 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\.os-homedir.DELETE'
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: os-tmpdir@1.0.2 (node_modules\fsevents\node_modules\os-tmpdir):
npm WARN enoent SKIPPING OPTIONAL DEPENDENCY: ENOENT: no such file or directory, rename 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\os-tmpdir' -> 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\.os-tmpdir.DELETE'
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: path-is-absolute@1.0.1 (node_modules\fsevents\node_modules\path-is-absolute):
npm WARN enoent SKIPPING OPTIONAL DEPENDENCY: ENOENT: no such file or directory, rename 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\path-is-absolute' -> 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\.path-is-absolute.DELETE'
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: process-nextick-args@2.0.1 (node_modules\fsevents\node_modules\process-nextick-args):
npm WARN enoent SKIPPING OPTIONAL DEPENDENCY: ENOENT: no such file or directory, rename 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\process-nextick-args' -> 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\.process-nextick-args.DELETE'
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: minimist@1.2.0 (node_modules\fsevents\node_modules\rc\node_modules\minimist):
npm WARN enoent SKIPPING OPTIONAL DEPENDENCY: ENOENT: no such file or directory, rename 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\rc\node_modules\minimist' -> 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\rc\node_modules\.minimist.DELETE'
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: safe-buffer@5.1.2 (node_modules\fsevents\node_modules\safe-buffer):
npm WARN enoent SKIPPING OPTIONAL DEPENDENCY: ENOENT: no such file or directory, rename 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\safe-buffer' -> 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\.safe-buffer.DELETE'npm WARN optional SKIPPING OPTIONAL DEPENDENCY: safer-buffer@2.1.2 (node_modules\fsevents\node_modules\safer-buffer):
npm WARN enoent SKIPPING OPTIONAL DEPENDENCY: ENOENT: no such file or directory, rename 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\safer-buffer' -> 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\.safer-buffer.DELETE'
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: sax@1.2.4 (node_modules\fsevents\node_modules\sax):
npm WARN enoent SKIPPING OPTIONAL DEPENDENCY: ENOENT: no such file or directory, rename 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\sax' -> 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\.sax.DELETE'
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: semver@5.7.1 (node_modules\fsevents\node_modules\semver):
npm WARN enoent SKIPPING OPTIONAL DEPENDENCY: ENOENT: no such file or directory, rename 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\semver' -> 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\.semver.DELETE'
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: set-blocking@2.0.0 (node_modules\fsevents\node_modules\set-blocking):
npm WARN enoent SKIPPING OPTIONAL DEPENDENCY: ENOENT: no such file or directory, rename 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\set-blocking' -> 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\.set-blocking.DELETE'
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: signal-exit@3.0.2 (node_modules\fsevents\node_modules\signal-exit):
npm WARN enoent SKIPPING OPTIONAL DEPENDENCY: ENOENT: no such file or directory, rename 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\signal-exit' -> 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\.signal-exit.DELETE'npm WARN optional SKIPPING OPTIONAL DEPENDENCY: strip-json-comments@2.0.1 (node_modules\fsevents\node_modules\strip-json-comments):
npm WARN enoent SKIPPING OPTIONAL DEPENDENCY: ENOENT: no such file or directory, rename 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\strip-json-comments' -> 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\.strip-json-comments.DELETE'
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: util-deprecate@1.0.2 (node_modules\fsevents\node_modules\util-deprecate):
npm WARN enoent SKIPPING OPTIONAL DEPENDENCY: ENOENT: no such file or directory, rename 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\util-deprecate' -> 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\.util-deprecate.DELETE'
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: wrappy@1.0.2 (node_modules\fsevents\node_modules\wrappy):
npm WARN enoent SKIPPING OPTIONAL DEPENDENCY: ENOENT: no such file or directory, rename 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\wrappy' -> 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\.wrappy.DELETE'
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: yallist@3.1.1 (node_modules\fsevents\node_modules\yallist):
npm WARN enoent SKIPPING OPTIONAL DEPENDENCY: ENOENT: no such file or directory, rename 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\yallist' -> 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\.yallist.DELETE'

added 601 packages from 421 contributors and audited 882446 packages in 28.723s

18 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities


C:\projects\hapi-cron>ls
'ls' is not recognized as an internal or external command,
operable program or batch file.

C:\projects\hapi-cron>dir
 Volume in drive C is speedy
 Volume Serial Number is 8CAE-3A4A

 Directory of C:\projects\hapi-cron

02/26/2020  10:06 PM    <DIR>          .
02/26/2020  10:06 PM    <DIR>          ..
02/26/2020  10:05 PM               257 .editorconfig
02/26/2020  10:05 PM                24 .eslintignore
02/26/2020  10:05 PM               327 .eslintrc
02/26/2020  10:05 PM               325 .gitignore
02/26/2020  10:05 PM                48 .travis.yml
02/26/2020  10:05 PM    <DIR>          lib
02/26/2020  10:05 PM             1,512 LICENSE
02/26/2020  10:06 PM    <DIR>          node_modules
02/26/2020  10:06 PM           226,439 package-lock.json
02/26/2020  10:05 PM             1,240 package.json
02/26/2020  10:05 PM             3,224 README.md
02/26/2020  10:05 PM    <DIR>          __tests__
               9 File(s)        233,396 bytes
               5 Dir(s)  118,307,016,704 bytes free

C:\projects\hapi-cron>cd __tests__

C:\projects\hapi-cron\__tests__>dir
 Volume in drive C is speedy
 Volume Serial Number is 8CAE-3A4A

 Directory of C:\projects\hapi-cron\__tests__

02/26/2020  10:05 PM    <DIR>          .
02/26/2020  10:05 PM    <DIR>          ..
02/26/2020  10:05 PM            10,920 index.js
               1 File(s)         10,920 bytes
               2 Dir(s)  118,319,104,000 bytes free

C:\projects\hapi-cron\__tests__>node index.js
C:\projects\hapi-cron\__tests__\index.js:13
describe('registration assertions', () => {
^

ReferenceError: describe is not defined
    at Object.<anonymous> (C:\projects\hapi-cron\__tests__\index.js:13:1)
[90m    at Module._compile (internal/modules/cjs/loader.js:1144:30)[39m
[90m    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1164:10)[39m
[90m    at Module.load (internal/modules/cjs/loader.js:993:32)[39m
[90m    at Function.Module._load (internal/modules/cjs/loader.js:892:14)[39m
[90m    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:71:12)[39m
[90m    at internal/main/run_main_module.js:17:47[39m

C:\projects\hapi-cron\__tests__>cd ..

C:\projects\hapi-cron>npm install --only=dev
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.11 (node_modules\fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.11: wanted {"os":"darwin","arch":"any"} (current: {"os":"win32","arch":"x64"})

audited 882446 packages in 7.871s

18 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities


C:\projects\hapi-cron>node __tests__/index.js
C:\projects\hapi-cron\__tests__\index.js:13
describe('registration assertions', () => {
^

ReferenceError: describe is not defined
    at Object.<anonymous> (C:\projects\hapi-cron\__tests__\index.js:13:1)
[90m    at Module._compile (internal/modules/cjs/loader.js:1144:30)[39m
[90m    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1164:10)[39m
[90m    at Module.load (internal/modules/cjs/loader.js:993:32)[39m
[90m    at Function.Module._load (internal/modules/cjs/loader.js:892:14)[39m
[90m    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:71:12)[39m
[90m    at internal/main/run_main_module.js:17:47[39m

C:\projects\hapi-cron>npm init --yes
Wrote to C:\projects\hapi-cron\package.json:

{
  "name": "hapi-cron",
  "version": "1.0.6",
  "description": "A Hapi plugin to setup cron jobs that will call predefined server routes at specified times",
  "main": "lib/index.js",
  "scripts": {
    "test": "npm run eslint && jest --runInBand",
    "eslint": "eslint ."
  },
  "repository": {
    "type": "git",
    "url": "git+https://github.com/antonsamper/hapi-cron.git"
  },
  "keywords": [
    "hapi",
    "plugin",
    "cron"
  ],
  "author": "Anton Samper Rivaya",
  "license": "BSD-3-Clause",
  "bugs": {
    "url": "https://github.com/antonsamper/hapi-cron/issues"
  },
  "homepage": "https://github.com/antonsamper/hapi-cron#readme",
  "devDependencies": {
    "eslint": "^6.0.0",
    "@hapi/eslint-config-hapi": "^12.1.0",
    "@hapi/eslint-plugin-hapi": "^4.3.3",
    "@hapi/hapi": "^18.3.1",
    "jest": "^24.8.0"
  },
  "dependencies": {
    "cron": "^1.7.1",
    "@hapi/hoek": "^7.2.0"
  },
  "engines": {
    "node": ">=8.0.0"
  },
  "jest": {
    "collectCoverage": true,
    "coverageThreshold": {
      "global": {
        "branches": 100,
        "functions": 100,
        "lines": 100,
        "statements": 100
      }
    },
    "testEnvironment": "node"
  },
  "directories": {
    "lib": "lib"
  }
}



C:\projects\hapi-cron>npm config get production
false

C:\projects\hapi-cron>npm install jest --save-dev
npm WARN deprecated left-pad@1.3.0: use String.prototype.padStart()
npm WARN deprecated request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@^1.2.7 (node_modules\jest-haste-map\node_modules\fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.11: wanted {"os":"darwin","arch":"any"} (current: {"os":"win32","arch":"x64"})
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: abbrev@1.1.1 (node_modules\fsevents\node_modules\abbrev):
npm WARN enoent SKIPPING OPTIONAL DEPENDENCY: ENOENT: no such file or directory, rename 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\abbrev' -> 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\.abbrev.DELETE'
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: ansi-regex@2.1.1 (node_modules\fsevents\node_modules\ansi-regex):
npm WARN enoent SKIPPING OPTIONAL DEPENDENCY: ENOENT: no such file or directory, rename 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\ansi-regex' -> 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\.ansi-regex.DELETE'
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: aproba@1.2.0 (node_modules\fsevents\node_modules\aproba):
npm WARN enoent SKIPPING OPTIONAL DEPENDENCY: ENOENT: no such file or directory, rename 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\aproba' -> 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\.aproba.DELETE'
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: balanced-match@1.0.0 (node_modules\fsevents\node_modules\balanced-match):
npm WARN enoent SKIPPING OPTIONAL DEPENDENCY: ENOENT: no such file or directory, rename 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\balanced-match' -> 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\.balanced-match.DELETE'
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: chownr@1.1.3 (node_modules\fsevents\node_modules\chownr):
npm WARN enoent SKIPPING OPTIONAL DEPENDENCY: ENOENT: no such file or directory, rename 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\chownr' -> 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\.chownr.DELETE'
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: code-point-at@1.1.0 (node_modules\fsevents\node_modules\code-point-at):
npm WARN enoent SKIPPING OPTIONAL DEPENDENCY: ENOENT: no such file or directory, rename 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\code-point-at' -> 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\.code-point-at.DELETE'
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: concat-map@0.0.1 (node_modules\fsevents\node_modules\concat-map):
npm WARN enoent SKIPPING OPTIONAL DEPENDENCY: ENOENT: no such file or directory, rename 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\concat-map' -> 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\.concat-map.DELETE'
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: console-control-strings@1.1.0 (node_modules\fsevents\node_modules\console-control-strings):
npm WARN enoent SKIPPING OPTIONAL DEPENDENCY: ENOENT: no such file or directory, rename 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\console-control-strings' -> 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\.console-control-strings.DELETE'
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: core-util-is@1.0.2 (node_modules\fsevents\node_modules\core-util-is):
npm WARN enoent SKIPPING OPTIONAL DEPENDENCY: ENOENT: no such file or directory, rename 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\core-util-is' -> 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\.core-util-is.DELETE'
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: deep-extend@0.6.0 (node_modules\fsevents\node_modules\deep-extend):
npm WARN enoent SKIPPING OPTIONAL DEPENDENCY: ENOENT: no such file or directory, rename 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\deep-extend' -> 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\.deep-extend.DELETE'
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: delegates@1.0.0 (node_modules\fsevents\node_modules\delegates):
npm WARN enoent SKIPPING OPTIONAL DEPENDENCY: ENOENT: no such file or directory, rename 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\delegates' -> 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\.delegates.DELETE'
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: detect-libc@1.0.3 (node_modules\fsevents\node_modules\detect-libc):
npm WARN enoent SKIPPING OPTIONAL DEPENDENCY: ENOENT: no such file or directory, rename 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\detect-libc' -> 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\.detect-libc.DELETE'
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fs.realpath@1.0.0 (node_modules\fsevents\node_modules\fs.realpath):
npm WARN enoent SKIPPING OPTIONAL DEPENDENCY: ENOENT: no such file or directory, rename 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\fs.realpath' -> 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\.fs.realpath.DELETE'
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: has-unicode@2.0.1 (node_modules\fsevents\node_modules\has-unicode):
npm WARN enoent SKIPPING OPTIONAL DEPENDENCY: ENOENT: no such file or directory, rename 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\has-unicode' -> 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\.has-unicode.DELETE'
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: inherits@2.0.4 (node_modules\fsevents\node_modules\inherits):
npm WARN enoent SKIPPING OPTIONAL DEPENDENCY: ENOENT: no such file or directory, rename 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\inherits' -> 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\.inherits.DELETE'
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: ini@1.3.5 (node_modules\fsevents\node_modules\ini):
npm WARN enoent SKIPPING OPTIONAL DEPENDENCY: ENOENT: no such file or directory, rename 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\ini' -> 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\.ini.DELETE'
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: isarray@1.0.0 (node_modules\fsevents\node_modules\isarray):
npm WARN enoent SKIPPING OPTIONAL DEPENDENCY: ENOENT: no such file or directory, rename 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\isarray' -> 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\.isarray.DELETE'
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: minimist@0.0.8 (node_modules\fsevents\node_modules\minimist):
npm WARN enoent SKIPPING OPTIONAL DEPENDENCY: ENOENT: no such file or directory, rename 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\minimist' -> 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\.minimist.DELETE'
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: ms@2.1.2 (node_modules\fsevents\node_modules\ms):
npm WARN enoent SKIPPING OPTIONAL DEPENDENCY: ENOENT: no such file or directory, rename 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\ms' -> 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\.ms.DELETE'
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: npm-normalize-package-bin@1.0.1 (node_modules\fsevents\node_modules\npm-normalize-package-bin):
npm WARN enoent SKIPPING OPTIONAL DEPENDENCY: ENOENT: no such file or directory, rename 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\npm-normalize-package-bin' -> 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\.npm-normalize-package-bin.DELETE'
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: number-is-nan@1.0.1 (node_modules\fsevents\node_modules\number-is-nan):
npm WARN enoent SKIPPING OPTIONAL DEPENDENCY: ENOENT: no such file or directory, rename 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\number-is-nan' -> 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\.number-is-nan.DELETE'
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: object-assign@4.1.1 (node_modules\fsevents\node_modules\object-assign):
npm WARN enoent SKIPPING OPTIONAL DEPENDENCY: ENOENT: no such file or directory, rename 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\object-assign' -> 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\.object-assign.DELETE'
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: os-homedir@1.0.2 (node_modules\fsevents\node_modules\os-homedir):
npm WARN enoent SKIPPING OPTIONAL DEPENDENCY: ENOENT: no such file or directory, rename 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\os-homedir' -> 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\.os-homedir.DELETE'
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: os-tmpdir@1.0.2 (node_modules\fsevents\node_modules\os-tmpdir):
npm WARN enoent SKIPPING OPTIONAL DEPENDENCY: ENOENT: no such file or directory, rename 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\os-tmpdir' -> 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\.os-tmpdir.DELETE'
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: path-is-absolute@1.0.1 (node_modules\fsevents\node_modules\path-is-absolute):
npm WARN enoent SKIPPING OPTIONAL DEPENDENCY: ENOENT: no such file or directory, rename 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\path-is-absolute' -> 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\.path-is-absolute.DELETE'
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: process-nextick-args@2.0.1 (node_modules\fsevents\node_modules\process-nextick-args):
npm WARN enoent SKIPPING OPTIONAL DEPENDENCY: ENOENT: no such file or directory, rename 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\process-nextick-args' -> 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\.process-nextick-args.DELETE'
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: minimist@1.2.0 (node_modules\fsevents\node_modules\rc\node_modules\minimist):
npm WARN enoent SKIPPING OPTIONAL DEPENDENCY: ENOENT: no such file or directory, rename 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\rc\node_modules\minimist' -> 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\rc\node_modules\.minimist.DELETE'
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: safe-buffer@5.1.2 (node_modules\fsevents\node_modules\safe-buffer):
npm WARN enoent SKIPPING OPTIONAL DEPENDENCY: ENOENT: no such file or directory, rename 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\safe-buffer' -> 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\.safe-buffer.DELETE'
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: safer-buffer@2.1.2 (node_modules\fsevents\node_modules\safer-buffer):
npm WARN enoent SKIPPING OPTIONAL DEPENDENCY: ENOENT: no such file or directory, rename 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\safer-buffer' -> 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\.safer-buffer.DELETE'
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: sax@1.2.4 (node_modules\fsevents\node_modules\sax):
npm WARN enoent SKIPPING OPTIONAL DEPENDENCY: ENOENT: no such file or directory, rename 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\sax' -> 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\.sax.DELETE'
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: semver@5.7.1 (node_modules\fsevents\node_modules\semver):
npm WARN enoent SKIPPING OPTIONAL DEPENDENCY: ENOENT: no such file or directory, rename 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\semver' -> 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\.semver.DELETE'
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: set-blocking@2.0.0 (node_modules\fsevents\node_modules\set-blocking):
npm WARN enoent SKIPPING OPTIONAL DEPENDENCY: ENOENT: no such file or directory, rename 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\set-blocking' -> 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\.set-blocking.DELETE'
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: signal-exit@3.0.2 (node_modules\fsevents\node_modules\signal-exit):
npm WARN enoent SKIPPING OPTIONAL DEPENDENCY: ENOENT: no such file or directory, rename 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\signal-exit' -> 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\.signal-exit.DELETE'
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: strip-json-comments@2.0.1 (node_modules\fsevents\node_modules\strip-json-comments):
npm WARN enoent SKIPPING OPTIONAL DEPENDENCY: ENOENT: no such file or directory, rename 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\strip-json-comments' -> 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\.strip-json-comments.DELETE'
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: util-deprecate@1.0.2 (node_modules\fsevents\node_modules\util-deprecate):
npm WARN enoent SKIPPING OPTIONAL DEPENDENCY: ENOENT: no such file or directory, rename 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\util-deprecate' -> 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\.util-deprecate.DELETE'
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: wrappy@1.0.2 (node_modules\fsevents\node_modules\wrappy):
npm WARN enoent SKIPPING OPTIONAL DEPENDENCY: ENOENT: no such file or directory, rename 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\wrappy' -> 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\.wrappy.DELETE'
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: yallist@3.1.1 (node_modules\fsevents\node_modules\yallist):
npm WARN enoent SKIPPING OPTIONAL DEPENDENCY: ENOENT: no such file or directory, rename 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\yallist' -> 'C:\projects\hapi-cron\node_modules\fsevents\node_modules\.yallist.DELETE'

+ jest@24.9.0
updated 1 package and audited 882446 packages in 23.195s

3 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities


C:\projects\hapi-cron>node __tests__/index.js
C:\projects\hapi-cron\__tests__\index.js:13
describe('registration assertions', () => {
^

ReferenceError: describe is not defined
    at Object.<anonymous> (C:\projects\hapi-cron\__tests__\index.js:13:1)
[90m    at Module._compile (internal/modules/cjs/loader.js:1144:30)[39m
[90m    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1164:10)[39m
[90m    at Module.load (internal/modules/cjs/loader.js:993:32)[39m
[90m    at Function.Module._load (internal/modules/cjs/loader.js:892:14)[39m
[90m    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:71:12)[39m
[90m    at internal/main/run_main_module.js:17:47[39m
```
💥  💥 💥 

Ok now my PR fork

```
C:\projects\hapi-cron>npm install
npm notice created a lockfile as package-lock.json. You should commit this file.
added 262 packages from 213 contributors and audited 684 packages in 13.272s

5 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

C:\projects\hapi-cron>node __tests__/index.js

  ..

2 tests complete
Test duration: 7 ms
Leaks: No issues

C:\projects\hapi-cron>
```
👏 👏 🕺 🎉 🎆 🍾 

The `jest` library is not installing and no I did not install it globally because I shouldn't have to it's listed as a project `devDependency`, but it's not working on my `MacOS X` and `Windows` machine, but I switch over to the cool `hapi` lab test system and bliss is achieved right from the start

antonsamper commented:
@Siliconrob Looking at your code above, this pr is invalid. Please correct me if I'm wrong but there are 2 things worth noting here:

1. What you describe above is cloning the repo as opposed to installing `hapi-cron` as a dependency.  Running `npm i hapi-cron` installs the module and the correct dependencies and the plugin works as expected. There are no issues there.
2. You are trying to run the tests by running `node index.js` inside of the `__tests__` folder. However, this isnt how you run the tests. Jest is a package, not a script. To run the tests you have a few options:
   - Install Jest globally as you mentioned above - This is not ideal but also not necessary as Jest gets installed as a dev dependency.
   - Simply run `npm t`, `npm test` or `npx jest` on the project root

Let me know if you have any questions.

Siliconrob commented:
I was doing what the dependency to my project for use, but it looks like there is some type of npm issue when various packages are being combined and your particular package flagged it out for me.

So that means I need to find the dependency conflict.  I will close the request and submit the updates to the supported dependencies as a new PR.  I verified this by making simple hapi project and only pulling in your dependency as the first outside `hapi` dependency.

As for why I pulled the project down and tried to test it was I wanted to find out what was going on and the log pointed to an issue with `hapi-cron` and then I got into trying to update and run your tests to verify that the updates I made are valid.
